### ISSUE TITLE
Gulp Build Issue

### ISSUE THREAD

**Issue by plieningerweb** [EESC-LabRoM/rosweb#5]
I did everything as described in the Contributor.md, but I get a issue:
```
gulp build
[23:42:45] Using gulpfile ~/projects/deeweeder/ros/rosweb/gulpfile.js
[23:42:45] Starting 'html'...
[23:42:45] Starting 'ts'...
[23:42:45] Starting 'sass'...
[23:42:45] Starting 'sass_wdgt'...
[23:42:45] Starting 'img'...
[23:42:45] Starting 'hbs'...
[23:42:45] Finished 'hbs' after 26 ms
[23:42:45] Starting 'js'...
[23:42:45] Starting 'wdgt'...
[23:42:45] Starting 'jsl_roslibjs'...
[23:42:45] Starting 'jsl_mjpegcanvasjs'...
[23:42:45] Starting 'jsl_ros3djs'...
[23:42:45] Starting 'jsl_eventemitter'...
[23:42:45] Starting 'jsl_handlebars'...
[23:42:45] Starting 'jsl_jquery'...
[23:42:45] Finished 'js' after 32 ms

events.js:160
      throw er; // Unhandled 'error' event
      ^
TypeScript error: ./src/ts/events/widget_instance.ts(187,14): Error TS2339: Property 'getActionServers' does not exist on type 'Ros'.
```

**Comment by marcoarruda** [comment]
Hi @plieningerweb,
Thanks for the note.
Actually, it is a temporary problem.

It is related to this pull request (https://github.com/RobotWebTools/roslibjs/pull/242), which includes a new method that I've created.

This method definition is missing on the definition file (src/typings/roslib/roslib.d.ts). For instance, I've added it manually:
`getActionServers(callback:(action_servers:string[]) => void, failedCallback?:(error:any)=>void):void;`

If the pull request be accepted, I'll be able to commit this definition to the DefinitelyTyped repo (https://github.com/DefinitelyTyped/DefinitelyTyped).
